src/pageEditor/store/editor/editorSelectors/editorModComponentSelectors.ts

```diff
@@ -75,11 +75,7 @@ export const selectActiveModComponentRef = createSelector(
 ///
 
 export const selectAllDeletedModComponentIds = ({ editor }: EditorRootState) =>
-  new Set(
-    flatMap(editor.deletedModComponentFormStatesByModId).map(
-      (formState) => formState.uuid,
-    ),
-  );
+  new Set(flatten(Object.values(editor.deletedModComponentFormStatesByModId)));
```

Comment:
NIT: why not use Array.flat()?

```suggestion
  new Set(Object.values(editor.deletedModComponentFormStatesByModId).flat());
```

Reply:
Because I ❤️ lodash